airbyte-integrations/connectors/source-confluence/source_confluence/manifest.yaml

```diff
@@ -121,12 +134,41 @@ definitions:
       name: "space"
       path: "space"
 
+  search_stream:
+    $ref: "#/definitions/base_stream"
+    retriever:
+      $ref: "#/definitions/base_retriever"
+      requester:
+        $ref: "#/definitions/requester"
+        url_base: https://{{ config['domain_name'] }}/wiki/
+        request_parameters:
+          cql: type=page {{  ("and space="  ~ config['space_name']) if config['space_name'] }} {{ ('and lastmodified >= "' ~ (stream_state.lastUpdated or config['start_date']) ~ '"') if stream_state.lastUpdated or config['start_date'] }}
+          expand: '["history.lastUpdated","body.view"]'
+    primary_key: "id"
+    $parameters:
+      name: "search"
+      path: /rest/api/content/search
+    transformations:
+      - "#/definitions/custom_field_transformation"
+    incremental_sync:
+      type: DatetimeBasedCursor
+      cursor_field: lastUpdated
+      cursor_datetime_formats:
+        - "%Y-%m-%dT%H:%M:%SZ"
+        - "%Y-%m-%dT%H:%M:%S.%fZ"
```

Comment:
Right now the stream is not working because the `lastUpdated` cursor isn't defined in the json schema.